dotcom-rendering/src/components/Card/Card.tsx

```diff
@@ -59,6 +59,16 @@ import { ImageWrapper } from './components/ImageWrapper';
 import { TrailText, type TrailTextSize } from './components/TrailText';
 
 export type Position = 'inner' | 'outer' | 'none';
+export const FAIRGROUND_CONTAINERS = [
```

Comment:
I'm wondering if there is a more standard name for this set of containers than "fairground". Happy for any suggestions!

Reply:
Yeah, otherwise we've been using "beta containers/collections", which obviously will require a rename at some point. Maybe best to use beta for now and when the time comes we can rename everything?

In terms of an actual name, hm. Other than maybe a temporal indication (erm... "TwentyTwentyFourRedesign" or something shorter?),  "Fairground" is probably the most meaningful option that jumps out (to me anyway).

Reply:
sounds like beta containers are the most standard so i'll rename for this 👍